packages/web/src/components/collection/desktop/CollectionHeader.js

```diff
@@ -626,7 +629,22 @@ class CollectionHeader extends PureComponent {
             <div className={cn(styles.description, fadeIn)}>
               <Linkify
                 options={{
-                  attributes: { onClick: onClickDescriptionExternalLink }
+                  attributes: {
+                    onClick: (event) => {
```

Comment:
We could make a helper function that we reuse for all of these instances of linkify. Something like `followLinkExternalOrInternal`? What do you think?

Reply:
i thought about this, but it seemed like more work than it was worth with the different sources for the tracking and the different function needed for the DM one